main/validators.py

```diff
@@ -0,0 +1,15 @@
+import re
```

Comment:
It provides a validator function which is used to check if the user enters a special character in the username or name.